sdk/mediaservices/Azure.Media.Analytics.Edge/README.md

```diff
@@ -0,0 +1,194 @@
+
+Use the client library for Live Video Analytics on IoT Edge to:
+
+- Simplify interactions with the [Microsoft Azure IoT SDKs](https://github.com/azure/azure-iot-sdks) 
+- Programatically construct media graph topologies and instances
+
+[Product documentation][doc_product] | [Direct methods][doc_direct_methods] | [Media graphs][doc_media_graph] | [Source code][source] | [Samples][samples]
+
+## Getting started
+### Authenticate the client
+n/a
+### Install the package
+
+Install the Live Video Analytics client library for .NET with NuGet:
+
+`dotnet add package Azure.Media.Analytics.Edge`
+
+### Prerequisites
+
+* C# is required to use this package.
+* You need an active [Azure subscription][azure_sub], and a [IoT device connection string][iot_device_connection_string] to use this package.
+* You will need to use the version of the SDK that corresponds to the version of the LVA Edge module you are using.
+
+    | SDK  | LVA Edge Module  |
+    |---|---|
+    | 1.0.0b1  | 2.0  |
+
+### Creating a graph topology and making requests
+Please visit the [Examples](#examples) for starter code
+## Key concepts
+
+### MediaGraph Topology vs MediaGraph Instance
+A _graph topology_ is a blueprint or template of a graph. It defines the parameters of the graph using placeholders as values for them. A _graph instance_ references a graph topology and specifies the parameters. This way you are able to have multiple graph instances referencing the same topology but with different values for parameters. For more information please visit [Media graph topologies and instances][doc_media_graph] 
+
+### CloudToDeviceMethod
+
+The `CloudToDeviceMethod` is part of the [azure-iot-hub SDk][iot-hub-sdk]. This method allows you to communicate one way notifications to a device in your IoT hub. In our case, we want to communicate various graph methods such as `MediaGraphTopologySetRequest` and `MediaGraphTopologyGetRequest`. To use `CloudToDeviceMethod` you need to pass in two parameters: `method_name` and `payload`. 
+
+The first parameter, `method_name`, is the name of the media graph request you are sending. Make sure to use each method's predefined `method_name` property. For example, `MediaGraphTopologySetRequest.method_name`. 
+
+The second parameter, `payload`, sends the entire serialization of the media graph request. For example, `MediaGraphTopologySetRequest.serialize()`
+
+## Examples
+
+### Creating a graph topology
+To create a graph topology you need to define parameters, sources, and sinks.
+```
```

Comment:
You should use snippets included from test code in your readmes.

Reply:
These are snippets from our sample code included under samples/LiveVideoAnalyticsSample.cs

Reply:
Right. We have a tool that make sure they are up-do-date with tests and are formatted correctly https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#updating-sample-snippets